diff --git a/src/client.js b/src/client.js
index 0ff29daa9ac1cd4f406944b3e195738d520b2c57..1bb5830f7a24e126e1ad2dd393d68ae25b29b621 100644
--- a/src/client.js
+++ b/src/client.js
@@ -518,6 +518,7 @@ class RPCClient extends EventEmitter {
     return this.request(RPCCommands.SET_ACTIVITY, {
       pid,
       activity: {
+        type: args.type,
         state: args.state,
         details: args.details,
         timestamps,
